tests/test_full_workflow.py

```diff
@@ -181,7 +181,7 @@ async def test_full_workflow(
 
         # Verify output
         res = await client.get(
-            f"{PREFIX}/dataset/{project_id}/{output_dataset_id}"
+            f"{PREFIX}/project/{project_id}/{output_dataset_id}"
```

Comment:
Note: this must be changed in the client as well: https://github.com/fractal-analytics-platform/fractal/issues/388